Makefile

```diff
@@ -91,15 +91,25 @@ $(VRAM_TAR): $(SPLASH) $(OVERRIDE_FONT) $(VRAM_DATA) $(VRAM_SCRIPTS)
 
 arm9/arm9.elf: $(VRAM_TAR)
 
-firm: $(ELF)
+$(OUTDIR)/AHBWRAM.bin: $(ELF)
+	@$(OBJCOPY) -O binary arm9/arm9.elf --only-section=AHBWRAM $@
+
+$(OUTDIR)/AHBWRAM2.bin: $(ELF)
+	@$(OBJCOPY) -O binary arm9/arm9.elf --only-section=AHBWRAM2 $@
+
+BINS := $(OUTDIR)/AHBWRAM.bin $(OUTDIR)/AHBWRAM2.bin
```

Comment:
Do these need to be .bin files? Objcopy should be able to extract only the section we care about while preserving LMA/VMA information as an .elf

Reply:
Seems to also work fine as an `.elf` file but it may take be a bit to figure out why objcopy is throwing a warning.